src/main/kotlin/model/Rank.kt

```diff
@@ -1,22 +1,33 @@
 package model
 
-import util.Constant
+import util.MoneyConstants
+
+private const val MATCH_COUNT_FIRST = 6
+private const val MATCH_COUNT_SECOND = 5
+private const val MATCH_COUNT_THIRD = 5
+private const val MATCH_COUNT_FOURTH = 4
+private const val MATCH_COUNT_FIFTH = 3
+private const val MATCH_COUNT_MISS = 0
 
 enum class Rank(val countOfMatch: Int, val winningAmount: Int) {
-    FIRST(6, Constant.FIRST_RANK_WINNING_MONEY),
-    SECOND(5, Constant.SECOND_RANK_WINNING_MONEY),
-    THIRD(5, Constant.THIRD_RANK_WINNING_MONEY),
-    FOURTH(4, Constant.FOURTH_RANK_WINNING_MONEY),
-    FIFTH(3, Constant.FIFTH_RANK_WINNING_MONEY),
-    MISS(0, Constant.MISS_WINNING_MONEY);
+    FIRST(MATCH_COUNT_FIRST, MoneyConstants.FIRST_RANK_WINNING_MONEY),
+    SECOND(MATCH_COUNT_SECOND, MoneyConstants.SECOND_RANK_WINNING_MONEY),
+    THIRD(MATCH_COUNT_THIRD, MoneyConstants.THIRD_RANK_WINNING_MONEY),
+    FOURTH(MATCH_COUNT_FOURTH, MoneyConstants.FOURTH_RANK_WINNING_MONEY),
+    FIFTH(MATCH_COUNT_FIFTH, MoneyConstants.FIFTH_RANK_WINNING_MONEY),
+    MISS(MATCH_COUNT_MISS, MoneyConstants.MISS_WINNING_MONEY),
```

Comment:
현재 단계에서는 상수만 보유하는 클래스가 불필요해 보입니다. 게다가 `util`에 등위별 총당첨금액을 넣어 두는 것이 적절할까요?

```suggestion
    FIRST(MATCH_COUNT_FIRST, 2_000_000_000),
    SECOND(MATCH_COUNT_SECOND, 30_000_000),
    THIRD(MATCH_COUNT_THIRD, 1_500_000),
    FOURTH(MATCH_COUNT_FOURTH, 50_000),
    FIFTH(MATCH_COUNT_FIFTH, 5_000),
    MISS(MATCH_COUNT_MISS, 0),
```